app/web/src/components/IdeHeader/IdeHeader.tsx

```diff
@@ -184,3 +117,96 @@ const IdeHeader = ({
 }
 
 export default IdeHeader
+
+function DefaultTopButtons({
```

Comment:
Nothing new in here, just moved into its own component for the override ability above.